iac/terraform/aws/cognito/cognito.tf

```diff
@@ -69,3 +74,17 @@ resource "aws_cognito_user_pool_client" "app_pool_client" {
   supported_identity_providers         = ["COGNITO"]
   explicit_auth_flows                  = ["ALLOW_USER_PASSWORD_AUTH", "ALLOW_REFRESH_TOKEN_AUTH", "ALLOW_USER_SRP_AUTH"]
 }
+
+resource "aws_cognito_user_group" "user_group" {
+  user_pool_id = aws_cognito_user_pool.user_pool.id
+  name         = "user"
+  description  = "Standard user group"
+  precedence   = 1
+}
+
+resource "aws_cognito_user_group" "organization_group" {
+  user_pool_id = aws_cognito_user_pool.user_pool.id
+  name         = "organization"
+  description  = "Organization user group"
+  precedence   = 0
```

Comment:
do we need `precedence`? I don't see any usage

Reply:
We don't need for now actually.

Reply:
remove that then, this can potentially create a bug later on

Reply:
removed from both places